src/katsdpmodels/rfi_mask.py

```diff
@@ -37,25 +37,29 @@ class RFIMask(models.SimpleHDF5Model):
     # https://github.com/python/mypy/issues/4717
 
     def is_masked(self, frequency: u.Quantity, baseline_length: u.Quantity,
-                  channel_bandwidth: u.Quantity = 0 * u.Hz) -> Any:
-        """Determine whether given frequency is masked for the given baseline length.
+                  channel_width: u.Quantity = 0 * u.Hz) -> Any:
```

Comment:
Thanks for catching this - I think I was undecided for a while and must have missed updating this one once I made up my mind.